app/serializers/snack_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class SnackSerializer < ActiveModel::Serializer
+  attributes :id, :name, :description, :snack_owner_email
+
+  def snack_owner_email
```

Comment:
```suggestion
  def owner_email
```

You can remove the model name prefix